addons/mass_mailing/models/mass_mailing.py

```diff
@@ -578,7 +578,25 @@ def _get_blacklist(self):
         blacklist = {}
         target = self.env[self.mailing_model_real]
         mail_field = 'email' if 'email' in target._fields else 'email_from'
-        if 'opt_out' in target._fields:
+        if self.mailing_model_real == "mail.mass_mailing.contact":
+            # If user is opt_out on One list but not on another or if two user
```

Comment:
Is this copied from Odoo code? If so, can be cherry-picked? If not, please put a reference as comment in the code to where this comes from and mentioning that this is OCB exclusive.

Reply:
No really, it wouldn't be possible to make a strict backport from v12 since this part has changed quite a bit. I took the general idea though, but the code stands for itself.

Reply:
I insist in putting the reference and saying in the comment OCB only

Reply:
Yeah, something like "inspired from $commit", so we can have a reference to consult.